sites/automationworld/server/templates/content/index.marko

```diff
@@ -68,7 +68,7 @@ $ const displayPublishedDate = ["event", "webinar", "contact"].includes(type) ?
               </if>
               <else-if(displayCarousel)>
                 $ const images = getAsArray(content, "images.edges").map(({ node }) => node);
-                <if(images.length > 1)>
+                <if(images.length > 1 && content.published < 1572411600000)>
```

Comment:
Use a Date object, I have no idea what `1572411600000` is without looking it up.
```js
<if(images.length > 1 && content.published < new Date(2019, 10, 30))>
```